mk/config.mk

```diff
@@ -1163,3 +1163,18 @@ $(eval $(call cfg-depends-all,CFG_WIDEVINE_PTA,CFG_DT CFG_WIDEVINE_HUK))
 # system (also called the target) to communicate with and use the I/O of the
 # host computer.
 CFG_SEMIHOSTING ?= n
+
+# Enable the semihosting console driver which inputs/outputs the characters
+# from/to the file or debug terminal on the semihosting host computer.
+# - Specify CFG_SEMIHOSTING_CONSOLE_FILE="{your_log_file}" to output the
+#   characters to that file. Output to "optee.log" by default.
```

Comment:
s/characters/trace messages/g    (lines 1167, 1170 & 1171)

Reply:
Changed in https://github.com/OP-TEE/optee_os/pull/6700/commits/4779839d926e158c5384ff3c9bb55b87df9bede5#diff-6301a2703b709daa4d84ead1c30bf67cdaf088abed68753a8f9c740f1e58b54a